.github/workflows/pr.yml

```diff
@@ -46,30 +47,31 @@ jobs:
             - '.github/workflows/ci.yml'
             - '.github/workflows/pr.yml'
 
-  call-buildsample-app-service-demo:
-    name: Build App Service Demo
-    needs: setupcheck
-    strategy:
-      fail-fast: true
-      matrix:
-        sampleName: ['AppServiceDemo']
-        configuration: ['Debug', 'Release']
-        platform: ['x86']
-    uses: ./.github/workflows/template-buildsample.yml
-    with:
-      sampleName:  ${{ matrix.sampleName }}
-      configuration: ${{ matrix.configuration }}
-      platform: ${{ matrix.platform }}
-      extraRunWindowsArgs: --no-deploy
-      skipBuild: ${{ needs.setupcheck.outputs.workflows != 'true' && needs.setupcheck.outputs.app-service-demo != 'true' }} # Skip if files haven't changed
+  # [Windows Issue #968]
```

Comment:
If this isn't working, fine, but please disable AppServiceDemo in its own PR first (both here in the PR yml, the CI upgrade yml, and the normal CI yml).